.stylelintrc.yaml

```diff
@@ -0,0 +1,8 @@
+    extends: "@cerner/stylelint-config-terra"
+    rules: 
+      scss/at-mixin-pattern: "^(terra-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$"
+      selector-no-qualifying-type: null
+      property-no-vendor-prefix: null
+      terra/custom-property-name: null
+      terra/custom-property-namespace: null
+      terra/custom-property-pseudo-selectors: null
```

Comment:
Can we remove these rule overrides?

Reply:
Do you need this file? It looks like it's also specified in the package.json

Reply:
These overrides were added by @AshishMotanamGurunadham to address some linter rule changes. We can sync up on them if you'd like them removed. 

I can remove the duplicate entries in `package.json`.

Reply:
e56c985de4eb413da56329bb62d281911c4981bd